test/test_nn.py

```diff
@@ -4916,6 +4916,51 @@ def softmarginloss_no_reduce_test():
         pickle=False)
 
 
+def multilabelsoftmarginloss_weights_test():
```

Comment:
I think you forgot to add this to `new_module_tests` (or `new_criterion_tests`, it shouldn't matter too much which one you put it in)